overlord/devicestate/devicemgr.go

```diff
@@ -259,14 +259,17 @@ func setClassicFallbackModel(st *state.State, device *auth.DeviceState) error {
 }
 
 func (m *DeviceManager) operatingMode() string {
+	if m.modeEnv.Mode == "" {
+		return "run"
+	}
 	return m.modeEnv.Mode
 }
 
 func (m *DeviceManager) ensureOperational() error {
 	m.state.Lock()
 	defer m.state.Unlock()
 
-	if m.operatingMode() == "install" {
+	if m.operatingMode() != "run" {
```

Comment:
Perhaps we should update the comment to also mention some other mode such as recover, to avoid the impression that the test considers that everything that's not run mode is install?

Reply:
Also we should add to the comment that this will indirectly also stop auto-refreshes